tests/scripts/pgocheck/pefile/ordlookup/__init__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from . import ws2_32
+from . import oleaut32
+
+'''
+A small module for keeping a database of ordinal to symbol
+mappings for DLLs which frequently get linked without symbolic
+infoz.
+'''
+
+ords = {
+    b'ws2_32.dll': ws2_32.ord_names,
```

Comment:
The pefile module is more general than than this task requires. It has functions to analyse portable executables in a variety of ways.